samples/images/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.png
```

Comment:
Let's not add these to .gitignore please; Since tornado will be used to do server-side rendering, we will have .png, .jpg, .jpeg files in here. If you need to restrict it, please restrict the .gitignore to only a single folder

Reply:
It's only in samples/images, so should not affect the other Tornado work

Reply:
Multiple .gitignore files might not be the way to go, but it is how I've done it in the past.

It will only ignore files in it's immediate directory, not parent directories.

https://riptutorial.com/git/example/2888/ignoring-files-in-subfolders--multiple-gitignore-files-